services/pdf-ms/templateExample/ekb-löpande.json

```diff
@@ -0,0 +1,480 @@
+{
+  "numberOfpages": 5,
+  "defaultFontSize": 11,
+  "texts":
+  [
+  {
+      "page": 0,
+      "x": 400,
+      "y": 685,
+      "size": 11,
+      "text": "{{valuesChanged}}",
+      "valueId": "valuesChanged"
+  },
+  {
+      "page": 0,
+      "x": 270,
+      "y": 648,
+      "size": 11,
+      "text": "{{otherMessage}}",
+      "maxWidthInChars": 50,
+      "valueId": "otherMessage"
+  },
+  {
+      "page": 0,
+      "x": 400,
+      "y": 458,
+      "size": 11,
+      "text": "{{user.personalNumber}}",
+      "valueId": "user.personalNumber"
```

Comment:
what do this do?

Reply:
It is used to check if the value has changed since last time or not, and if so, change it to have a red color. Unsure where to properly document this.

Reply:
Can you name it to that then? last changed ...someting